### PR TITLE
Update installation

### PR DESCRIPTION
**Pull request by hanserasmus**
Just separated it so the user can easily click the copy button, and only be presented with the command.

### CODE REVIEW

File: docs/installation/3-netbox.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ Next, clone the **master** branch of the NetBox GitHub repository into the curre
 
 ```no-highlight
 $ sudo git clone -b master https://github.com/netbox-community/netbox.git .
+```
+The screen below should be the result:
```
Comment (jeremystretch) [inline docs/installation/3-netbox.md:77]: Please add newlines above and below this text to remain consistent with the rest of the document.

**Comment by jeremystretch** [comment]
@hanserasmus are you still interested in pursuing this PR?

File: docs/installation/3-netbox.md (hunk 1 above)
Comment (hanserasmus) [reply docs/installation/3-netbox.md:78]: Added.

**Comment by hanserasmus** [comment]
@jeremystretch Sorry for the delay. I forgot about this one completely and it got lost in the mail. Apologies.

**Pull request merged**